.circleci/config.yml

```diff
@@ -2,6 +2,17 @@ version: 2.1
 
 orbs:
   hmpps: ministryofjustice/hmpps@3.2
+  jira: circleci/jira@1.2.2
+
+executors:
+  deployer:
```

Comment:
We shouldn't need this I don't think if we're using `hmpps/deploy_env`

Reply:
Removed